src/library/scala/collection/mutable/Buffer.scala

```diff
@@ -19,7 +19,19 @@ trait Buffer[A]
     */
   def prepend(elem: A): this.type
 
-  @`inline` final def append(elem: A): this.type = addOne(elem)
+  /** Appends the given elements to this buffer.
+    *
+    *  @param elems  the elements to append.
+    */
+  @deprecated("Use addAll instead", "2.13.0")
+  def append(elems: A*) { appendAll(elems) }
```

Comment:
I suggest to not use the procedure syntax (which has been deprecated), and to forward to `addAll` directly:

~~~ scala
def append(elems: A*): Unit = addAll(elems)
def appendAll(xs: IterableOnce[A]): Unit = addAll(xs)
~~~

Reply:
updated. thanks.